apps/web/src/components/tx/SponsoredBy/styles.module.css

```
@@ -17,4 +17,15 @@
 .logo {
   width: 16px;
   height: 16px;
+  margin-left: 4px;
```

The gap is a bit too big now, let's try 2px.

```suggestion
  margin-left: 2px;
```